import_map.json

```diff
@@ -1,19 +1,15 @@
 {
   "imports": {
-    "polkadot/": "http://localhost:4646/frame/wss/rpc.polkadot.io/@latest/",
-    "polkadot_dev/": "http://localhost:4646/frame/dev/polkadot/@latest/",
-    "kusama/": "http://localhost:4646/frame/wss/kusama-rpc.polkadot.io/@latest/",
-    "kusama_dev/": "http://localhost:4646/frame/dev/kusama/@latest/",
-    "westend/": "http://localhost:4646/frame/wss/westend-rpc.polkadot.io/@latest/",
-    "westend_dev/": "http://localhost:4646/frame/dev/westend/@latest/",
-    "rococo/": "http://localhost:4646/frame/wss/rococo-contracts-rpc.polkadot.io/@latest/",
-    "rococo_contracts/": "http://localhost:4646/frame/wss/rococo-contracts-rpc.polkadot.io/@latest/",
-    "rococo_dev/": "http://localhost:4646/frame/dev/rococo/@latest/",
-    "acala/": "http://localhost:4646/frame/wss/acala-polkadot.api.onfinality.io/public-ws/@latest/",
-    "moonbeam/": "http://localhost:4646/frame/wss/wss.api.moonbeam.network/@latest/",
-    "statemint/": "http://localhost:4646/frame/wss/statemint-rpc.polkadot.io/@latest/",
-    "phala/": "http://localhost:4646/frame/wss/api.phala.network/ws#/rpc/@latest/",
-    "subsocial/": "http://localhost:4646/frame/para.subsocial.network/@latest/"
+    "polkadot/": "http://localhost:4646/frame/wss/rpc.polkadot.io/@v0.9.39/",
+    "polkadot_dev/": "http://localhost:4646/frame/dev/polkadot/@v0.9.36/",
```

Comment:
For CI, the dev versions might need to match 
https://github.com/paritytech/capi/blob/3da4f2edf7c4f06b4a39f26a64e8faec7ff6f220/.github/actions/setup-binaries/action.yml#L6-L17